digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -655,9 +655,9 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService, implici
         geometryWKTForPoints(railwayCrossing.lon, railwayCrossing.lat),
         "linkId" -> railwayCrossing.linkId,
         "m_value" -> railwayCrossing.mValue,
-        "safetyEquipment" -> railwayCrossing.propertyData.find(_.publicId == "turvavarustus").get.values.map(_.asInstanceOf[PropertyValue]).headOption.get.propertyValue.orElse(""),
-        "name" -> railwayCrossing.propertyData.find(_.publicId == "rautatien_tasoristeyksen_nimi").get.values.map(_.asInstanceOf[PropertyValue]).headOption.get.propertyValue.orElse(""),
-        "railwayCrossingId" -> railwayCrossing.propertyData.find(_.publicId == "tasoristeystunnus").get.values.map(_.asInstanceOf[PropertyValue]).headOption.get.propertyValue.orElse(""),
+        "safetyEquipment" -> railwayCrossing.propertyData.find(_.publicId == "turvavarustus").get.values.map(_.asInstanceOf[PropertyValue]).headOption.map(_.propertyValue).getOrElse(""),
+        "name" -> railwayCrossing.propertyData.find(_.publicId == "rautatien_tasoristeyksen_nimi").get.values.map(_.asInstanceOf[PropertyValue]).headOption.map(_.propertyValue).getOrElse(""),
+        "railwayCrossingId" -> railwayCrossing.propertyData.find(_.publicId == "tasoristeystunnus").get.values.map(_.asInstanceOf[PropertyValue]).headOption.map(_.propertyValue).getOrElse(""),
```

Comment:
I did a suggestion related with this on US 2262 that I will apply here.
Suggestion put this get property on a function inside this ToApi:
```scala
def getProperty(crossing: RailwayCrossing, publicId: String) = {
crossing..propertyData.find(_.publicId == "publicId").get.values.map(_.asInstanceOf[PropertyValue]).headOption.map(_.propertyValue).getOrElse("")
}

...
examples

          "turvavarustus" -> getProperty(railwayCrossing, "turvavarustus"),
          "railwayCrossingId" -> getProperty(railwayCrossing, "railwayCrossingId")

```

Reply:
resolved by using each asset's service